docs/modules/ROOT/pages/release-notes.adoc

```diff
@@ -1,39 +1,34 @@
+* Added WAN Replication (https://github.com/hazelcast/hazelcast-platform-operator/pull/226[#226])
+* Added EntryProcessor support (https://github.com/hazelcast/hazelcast-platform-operator/pull/334[#334])
+* Added MapStore/MapLoader support (https://github.com/hazelcast/hazelcast-platform-operator/pull/291[#291])
+* Added Executor Service support (https://github.com/hazelcast/hazelcast-platform-operator/pull/300[#300])
 
 == Changes
 
-* Added Hazelcast members detailed status (https://github.com/hazelcast/hazelcast-platform-operator/pull/171[#171])
-* Added HotBackup status (https://github.com/hazelcast/hazelcast-platform-operator/pull/179[#179])
-* Added periodic member list and status update (https://github.com/hazelcast/hazelcast-platform-operator/pull/180[#180])
-* Added client connection message to Hazelcast CR status (https://github.com/hazelcast/hazelcast-platform-operator/pull/176[#176])
-* Added restore status Hazelcast CR (https://github.com/hazelcast/hazelcast-platform-operator/pull/187[#187])
-* Added HotBackup pending status (https://github.com/hazelcast/hazelcast-platform-operator/pull/205[#205])
-* Implemented map persistence using Hazelcast ConfigMap (https://github.com/hazelcast/hazelcast-platform-operator/pull/204[#204])
-* Defined restore agent config types (https://github.com/hazelcast/hazelcast-platform-operator/pull/196[#196])
-* Introduced `internal` package (https://github.com/hazelcast/hazelcast-platform-operator/pull/215[#215])
-* Refactored logger usage and removed unnecessary logging (https://github.com/hazelcast/hazelcast-platform-operator/pull/213[#213])
-* Use different logger options for local and production environments (https://github.com/hazelcast/hazelcast-platform-operator/pull/214[#214])
-* Set HotBackup to failed when Hazelcast CR does not exist (https://github.com/hazelcast/hazelcast-platform-operator/pull/222[#222])
-* Made `resources` and `scheduling` fields updatable for Hazelcast and MC CRs (https://github.com/hazelcast/hazelcast-platform-operator/pull/227[#227])
-* Improved client to cluster connection check (https://github.com/hazelcast/hazelcast-platform-operator/pull/209[#209])
-* Added GS and Azure support for backup and restore (https://github.com/hazelcast/hazelcast-platform-operator/pull/230[#230])
-* Upgraded Hazelcast and MC versions to 5.1.2 and 5.1.3 (https://github.com/hazelcast/hazelcast-platform-operator/pull/241[#241])
+* Added user code deployment possibility from the client side (https://github.com/hazelcast/hazelcast-platform-operator/pull/340[#340])
+* Added uploading JAR's via ConfigMap (https://github.com/hazelcast/hazelcast-platform-operator/pull/304[#304])
+* Added downloading JAR into classpath (PVC) functionality (https://github.com/hazelcast/hazelcast-platform-operator/pull/244[#244])
+* Added enabling Map WAN Reference (https://github.com/hazelcast/hazelcast-platform-operator/pull/216[#216])
```

Comment:
```suggestion
* Enabled a reference to the `Map` resource so that the resource is updated with configuration changes for WAN replication (https://github.com/hazelcast/hazelcast-platform-operator/pull/216[#216])
```
Not sure I've got this right....